scripts/monitoring/cron-send-logging-checks.py

```diff
@@ -344,11 +344,18 @@ def run(self):
         self.get_pods()
 
         logging_status = {}
-        logging_status['elasticsearch'] = self.check_elasticsearch()
-        logging_status['fluentd'] = self.check_fluentd()
-        logging_status['kibana'] = self.check_kibana()
+        try:
+            logging_status['elasticsearch'] = self.check_elasticsearch()
+            logging_status['fluentd'] = self.check_fluentd()
+            logging_status['kibana'] = self.check_kibana()
+            self.report_to_zabbix(logging_status)
+        except:
+            result = 0 #something wrong with the checking script
+            mts = MetricSender(verbose=self.args.verbose)
+            mts.add_metric({'openshift.master.logging.elasticsearch.script.status': result})
```

Comment:
new metrics item? 0 stands for bad ? if there is no exception, should we send the metrics ? and what's the value?

Reply:
its dynamic ones , too complex , its far more easy to create a new one